elementalconductor/job.go

```diff
@@ -178,13 +201,19 @@ type FileGroupSettings struct {
 	Destination Location `xml:"destination,omitempty"`
 }
 
+// AppleLiveGroupSettings define where the HLS job output should go
+type AppleLiveGroupSettings struct {
+	Destination Location `xml:"destination,omitempty"`
+}
```

Comment:
They have the same content for now, but they are different nodes in the XML. Also the AppleLiveGroupSettings have lots of more options, according to the Elemental documentation I did not implement for now since I didn’t need it.


It’s a close call. I’ll follow the path of least resistance and leave it like this since it’s very likely we’ll have to add HLS settings later… or we finish the work on the wrapper and add the documented additional settings.